index.js

```diff
@@ -80,5 +83,16 @@ module.exports = {
 
   _readTemplate: function(name) {
     return fs.readFileSync(path.join(__dirname, 'templates', name + '.html'));
+  },
+
+  postprocessTree: function(type, tree) {
+    if (type === 'lint-app' || type === 'lint-tests' || type === 'lint-addon') {
+      return jshintTrees(tree, {
+        jshintrcPath: this.jshintrc.tests,
+        description: 'JSHint - QUnit'
```

Comment:
Yes, the point is the type. When this shows up in the slow trees log how do we know WHICH tree was slow?